strings/accents/accent_skaven.json

```diff
@@ -0,0 +1,294 @@
+{
+
+    "Skaven": {
+
+
+
+		
+		"captain":"captain-thing",
+		"head":"head-thing",
```

Comment:
If someone said "Head to Cargo" that could get confusing. also prefer something like "boss-boss" or "boss-master"